examples/scripts/blender/sdf_exporter.py

```diff
@@ -22,7 +22,7 @@ def export_sdf(prefix_path):
     meshes_folder_prefix = 'meshes/'
     
     # Exports the dae file and its associated textures
-    bpy.ops.wm.collada_export(filepath=prefix_path+meshes_folder_prefix+dae_filename, check_existing=False, filter_blender=False, filter_image=False, filter_movie=False, filter_python=False, filter_font=False, filter_sound=False, filter_text=False, filter_btx=False, filter_collada=True, filter_folder=True, filemode=8)
+    bpy.ops.wm.collada_export(filepath=os.path.join(prefix_path, meshes_folder_prefix, dae_filename), check_existing=False, filter_blender=False, filter_image=False, filter_movie=False, filter_python=False, filter_font=False, filter_sound=False, filter_text=False, filter_btx=False, filter_collada=True, filter_folder=True, filemode=8)
```

Comment:
Would be good to break this long line into 80-character lines.

Reply:
I formatted everything via the last commit https://github.com/gazebosim/gz-sim/pull/1718/commits/e17968bfe942ac466954d51c54d8bd9bda448cae using `isort` and `black` with default settings.

Originally, I did not want to do this because of the number of changes this generates (although this commit fixes a lot of things like quote and indentation inconsistency). Please, let me know if I should revert it and fix only the line lengths.